app/com/gu/contentapi/sanity/package.scala

```diff
@@ -23,10 +23,11 @@ package object sanity extends ScalaFutures with Matchers with IntegrationPatienc
   def incrementIncidentCount = {
     incidentCount += 1
```

Comment:
I have a feeling that the vars (`incidentKeyDateTime`, etc.) keeping track of the failure counts are shared across all tests, so the following situation would cause a PagerDuty alert to be triggered. Is that expected?

1. Test A fails
2. 9 minutes later, Test B fails
3. 9 minutes after that, Test C fails